### PR TITLE
pda clean ops

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

cleans some pda stuff
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by LightVillet** [comment]
Making medical PDA children of intern's PDA looks like coconut tf2 moment

**Comment by Rainbeon** [comment]
Given Intern roles are supposedly on the chopping block, I'd agree it seems like a bad idea to parent them to the Intern's one specifically

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml [hunk 1]
```diff
@@ -279,7 +279,7 @@
   - type: Pda
     id: MimeIDCard
     state: pda-mime
-    idSlot:
+    idSlot: #  rewrite without sound because mime
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Devices/pda.yml:282]: pretty sure you can just set the sound datafield to null

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml [hunk 2]
```diff
@@ -893,7 +870,7 @@
     state: pda-detective
 
 - type: entity
-  parent: BasePDA
+  parent: MedicalInternPDA
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Devices/pda.yml:873]: make a baseMedicalPDA and then use composition to have 2 parents.

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml (hunk 1 above)
Comment (lzk228) [reply Resources/Prototypes/Entities/Objects/Devices/pda.yml:282]: i looked and thought "why is it writted again?"
so its just a note for that ones who will think the same

**Review by EmoGarbage404 — APPROVED** [review]

**Pull request merged**